### ISSUE TITLE
Dockerfile gives multiple errors.

### ISSUE THREAD

**Issue by FeroWar** [nitrog0d/palworld-arm64#9]
So basically while trying to set it up i couldnt get it to work so i ran the dockerfile by doing "sudo docker build -t server ." and after a while it game me errors in the building process seen bellow.
i dont really know what i am doing so any help is apreciated.
![imagem](https://github.com/nitrog0d/palworld-arm64/assets/104322456/b03ea4d8-7f1b-4419-8d51-338edec7e0ad)

**Comment by Rodelkoenig** [comment]
Did you chmod 777 / chown your bind volume?

**Comment by FeroWar** [comment]
again as i said i am very new to docker and linux servers in general can you explain where or what the bind volume is.
i alredi did chmod 777 for the folder docker is started in and for palworld.

**Comment by Rodelkoenig** [comment]
![image](https://github.com/nitrog0d/palworld-arm64/assets/49457718/2e436a04-a6d1-472f-93a6-b5aa74da4d4c)

This refers to step 2 of the "Getting started":

> Create palworld sub-directory on the folder and run chmod 777 palworld for full permissions or use chown -R 1000:1000 palworld/.

Do you have this palworld folder in "/home/ubuntu/palworld-arm64" and did you chmod 777 it?

**Comment by FeroWar** [comment]
yes i do and i did both commands and confirmed that its permisions are 777.
